### ISSUE TITLE
Please create a tag for Coq 8.17 in Coq Platform 2023.03

### ISSUE THREAD

**Issue by MSoegtropIMC** [coq-community/math-classes#115]
The Coq team released Coq `8.17+rc1` on December 29th, 2022 and plans to release Coq 8.17.0 around March 7th, 2023.
The corresponding Coq Platform release `2023.03` should be released before **April 14th, 2023**.
It can be delayed in case of difficulties until June 30, 2023, but this should be an exception.

This issue is to inform you that to our (possibly a few days old) best knowledge the latest released version of your project (8.15.0) **does not work** with Coq `8.17+rc1`.
We tried to remove version restrictions in opam files and possibly make or configure files, but this did not suffice.

Please note that in Coq Platform CI (unlike Coq CI) we test only released / tagged versions. Coq CI is currently testing commit 754ddc768299afdf6274d03d713907d4d0282361 on repository https://github.com/coq-community/math-classes - which likely means that this commit does work in Coq CI.

Could you please create a tag and opam package, or communicate us any existing tag that works with Coq branch v8.17, **preferably before March 21st, 2023**?
In case we might have to delay the Coq Platform release cause of issues with your project, we would prefer to be informed about the situation as early as possible.

In case the tag and opam package are available before March 21st, 2023, it will be included in an early Coq Platform beta release of the for Coq 8.17+rc1.

The working branch of Coq Platform, can be found here [main](https://github.com/coq/platform/tree/main).
It contains package pick [`~8.17~2023.03+preview1`](https://github.com/coq/platform/tree/main/package_picks/package-pick-8.17~2023.03+preview1.sh) which already supports Coq version `8.17+rc1` and contains already working (possibly patched / commit pinned) Coq Platform packages.

Please **don't** close this issue, even after creating the new tag and/or opam package.
We will close the issue after updating Coq Platform.
This is a measure of 'double book keeping' in order to avoid that we ship the wrong version.

In any case, Coq Platform won't be released before this issue is closed!

Thanks!

P.S.: this issue has been created automatically based on CI status.

CC: https://github.com/coq/platform/issues/335

**Comment by MSoegtropIMC** [comment]
Yes - as stated in the message Coq CI seems to work. Coq Platform CI tests only releases / tags, though, and there is no tag which works. So what I would need is a tag.

**Comment by Zimmi48** [comment]
@spitters FWIW, the PR you cite is irrelevant (it is for compatibility with Coq 8.18). Anyway, I can propose a PR to test the master branch with Coq 8.17 and create a release.

**Comment by spitters** [comment]
Thanks!

**Comment by MSoegtropIMC** [comment]
Thanks! I updated the 2023.03 / 8.17 preview pick in my preparation branch to version 8.17.0 and close this issue.

The opam PR is here: https://github.com/coq/opam-coq-archive/pull/2506
